database.go

```diff
@@ -1049,6 +1049,35 @@ func (rp *RetentionPolicy) shardGroupByID(shardID uint64) *ShardGroup {
 	return nil
 }
 
+// dropMeasurement will remove a measurement from the index.
+func (db *database) dropMeasurement(name string) error {
```

Comment:
Oh, hang on, this is metastore updates? OK, yeah, that will required finer deletions I think (need to check the source). But removing the actual Measurement data? I thought that was just a bucket deletion.